### PR TITLE
chore: Configure release please properly

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

When I initially added release-please based off of the Netlify Next.js plugin, I didn't realise that the configuration had to be different. I believe this is the correct configuration.

## Related Tickets & Documents

https://github.com/netlify/pod-ecosystem-frameworks/issues/84

### CODE REVIEW

File: .release-please-manifest.json [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
```
Comment (nickytonline) [inline .release-please-manifest.json:1]: I added this when things weren't working after the initial addition of release-please, but I see it's not required, see https://github.com/netlify/vite-plugin-netlify-edge.

File: .github/workflows/release-please.yml [hunk 2]
```diff
@@ -16,27 +16,21 @@ jobs:
...
+      - uses: actions/checkout@v2
+        if: ${{ steps.release.outputs.release_created }}
+      - uses: actions/setup-node@v2
         with:
           node-version: '*'
           cache: 'npm'
           check-latest: true
           registry-url: 'https://registry.npmjs.org'
-        if: ${{ steps.release.outputs.releases_created }}
+        if: ${{ steps.release.outputs.release_created }}
```
Comment (nickytonline) [inline .github/workflows/release-please.yml:29]: I'm copying this from https://github.com/netlify/vite-plugin-netlify-edge @ascorbic, so I'm assuming it's `release_created` and not `releases_created`?

Comment (ascorbic) [reply .github/workflows/release-please.yml:29]: Yes, it's `releases_created` if using a manifest to handle releases, and `release_created` if using the default

Comment (nickytonline) [reply .github/workflows/release-please.yml:29]: Sounds good. I removed the manifest in this PR as it doesn't seem to be required based on https://github.com/netlify/vite-plugin-netlify-edge, so this should be good to merge then.

**Comment by nickytonline** [comment]
And it works!

CleanShot 2022-05-10 at 10 00 02

**Pull request merged**